src/Focusable.ts

```diff
@@ -378,35 +395,32 @@ export class FocusableAPI implements Types.FocusableAPI {
             return NodeFilter.FILTER_SKIP;
         }
 
-        if (state.ignoreUncontrolled) {
-            if (shouldIgnoreFocus(element)) {
-                return NodeFilter.FILTER_SKIP;
+        if (shouldIgnoreFocus(element)) {
+            if (
+                this._tabster.focusable.isFocusable(
```

Comment:
```suggestion
                this.isFocusable(
```

This is a member of the same class, why do you need to accecss it through a tabster proxy?